remotespark/remotesparkmagics.py

```diff
@@ -99,18 +110,29 @@ def spark(self, line, cell="", local_ns={}):
         # info
         if subcommand == "info":
             self._print_info()
+        # config
+        elif subcommand == "config":
+            # Would normally do " ".join(args.command[1:]) but parse_argstring removes quotes...
+            rest_of_line = user_input[7:]
+            self.properties = json.loads(rest_of_line)
```

Comment:
So it looks like the design we have here is that `self.properties` is the "source of truth" for configurations, unlike other configurations where the functions that come from the configuration.py module are the "source of truth".  I don't necessarily think this is an issue right now which is why I +1'd but it's something to keep in mind for the future.